shadow-cljs.edn

```diff
@@ -0,0 +1,26 @@
+{:lein {:profile "+shadow"}
+ ;; tests are served via http://localhost:8021
+ :dev-http {8021 "target/shadow-browser-test"}
+
+ :builds
+ {:node-test
+  {:target    :node-test
+   :output-to "target/shadow-node-test/node-tests.js"
+   :ns-regexp "-test"
+   ;; Watch will also run the tests
+   :autorun   true
+   :compiler-options {:warnings {:redef-in-file false
+                                 :fn-deprecated false}}}
+
+  :browser-test
```

Comment:
unused in CI, do we need it?

Reply:
It is the fastest/easiest way to run in browser locally, without Karma